test/passes/simplify-locals_all-features.txt

```diff
@@ -161,14 +161,18 @@
    (drop
     (i32.const 8)
    )
-   (nop)
-   (nop)
+   (local.set $a
```

Comment:
The reason this file has changed is because this runs with `--all-features`. (This test file requires at least atomics and reference types.) Should we add `--disable-exception-handling` or something to preserve the output?

Reply:
To preserve the intentions of existing tests, I made a copy of this file as simplify-locals_all-features_disable-exception-handling.wast.

Reply:
Sounds good, and was that file's output identical to what it was before?

Reply:
Yup